scipy/special/tests/test_pdtr.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import scipy.special as sc
+import pytest
+from numpy.testing import assert_almost_equal, assert_array_equal
+from numpy import isnan
+
+
+class TestCephes(object):
```

Comment:
Split this into two classes: `TestPdtr` and `TestPdtrc`. Then you can also shorten the test names (i.e. `test_pdtr_rounding` -> `test_rounding`).

Reply:
done